src/sentry/interfaces/stacktrace.py

```diff
@@ -859,6 +854,7 @@ def get_hash(self, platform=None, system_frames=True):
         if stack_invalid:
             return []
 
+        system_frames = variant == 'system'
         if not system_frames:
```

Comment:
Might be a little clearer to collapse these lines rather than have the assignment only used in the conditional.